GUIDELINES_AND_INFO.md

```diff
@@ -0,0 +1,20 @@
+# ScratchVerifier Information
+## What this is and is not
+This is not:
+- a platform for Scratchers to use except as an API for programs
+- a website that allows you to link accounts, or a social media
+
+This is
+- an API for Scratch account verification
+
+## Guidelines
+(NOT DONE)
+Please read our [Code of Conduct](https://github.com/ScratchVerifier/ScratchVerifier/blob/develop/CODE_OF_CONDUCT.md) before proceeding. 
+The following behaviour is not encouraged:
+- Posting issues with very simple descriptions, or unconstructive criticism. We would like you to in detail explain what can be done, what is wrong and what could get an update.
+- Irrelevant discussions: Please move to our Gitter channel off-topic (coming soon) to continue any discussions. Any long off-topic discussions could make the discussion get marked as offtopic, removed or the issue locked outright if the problem is worse enough
+- PRs with nearly no effort: We want to make quality code, so please submit a PR that has work in it. Copying code is fine, (everyone does that) but we will not accept any changes which do not provide any benefit to the project or its users.
```

Comment:
Maybe add to clearly state in the PR what benefits the additions/changes have, or encourage contributers to link PRs to open issues.

Reply:
kind of done, reopen if not